bin/lib/zopen-install

```diff
@@ -114,14 +115,21 @@ installDependencies()
   if [ "$dependencies" != "No dependencies" ]; then
     printHeader "Installing dependencies for $name: $dependencies..."
     
+    # Remove an old one if it exists
+    rm -f "${name}/.depsenv"
```

Comment:
since you have an rm -f here, do you want to protect this with a check it isn't empty and fail?

Reply:
I decided not to fail if it's not empty because we could be reinstalling the port. So I forcefully remove it (which won't complain if it doesn't exist).